docs/content/overview/product-telemetry.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Product Telemetry"
+description: "Learn how and why NGINX Ingress Controller collects product telemetry data."
+weight: 500
+toc: true
+---
+
+## Overview
+
+NGINX Ingress Controller collects product telemetry data to allow its developers to understand how it's deployed and configured by users.
+The goal is to use this data to triage development work and prioritize features and functionality that will provide the most value to our users.
+
+## About Product Telemetry
+
+Product telemetry is collected by default.
+Data is collected once every 24 hours, and is sent to a service managed by F5 over HTTPS.
+Personally identifiable information (PII) is **not** collected. 
```

Comment:
```suggestion
Product telemetry is enabled by default, collected once every 24 hours. It's then sent to a service managed by F5 over HTTPS.
```

Reducing the paragraphs: each sentence is relatively short. It's jarring having PII mentioned because it doesn't feel relevant in this case: it would be a concern for a user-facing application such as a mobile app, not an infrastructure tool.